README.md

```diff
@@ -1,5 +1,7 @@
 assignment_markup_warmup_sprint
 ===============================
 
+Julia Herron
+
 And 1 and 2!  And 1, and 2...!
-[An HTM5, CSS3 and SASS project from the Viking Code School](http://www.vikingcodeschool.com)
+[An HTM6, CSS3 and SASS project from the Viking Code School](http://www.vikingcodeschool.com)
```

Comment:
Whoops, not sure how that got changed! It's HTML5 :stuck_out_tongue_closed_eyes: